packages/twenty-server/src/engine/core-modules/auth/strategies/microsoft.auth.strategy.ts

```diff
@@ -0,0 +1,66 @@
+import { PassportStrategy } from '@nestjs/passport';
+
+import { Request } from 'express';
+import { VerifyCallback } from 'passport-google-oauth20';
```

Comment:
check that google import is relevant

Reply:
Yes, since the `passport-microsoft` doesn't provide `VerifyCallback` export. we need to use a type for a callback in validate function in this Strategy implementation. Since for google and microsoft the callback type will be the same, I guess we can use the type type from `passport-google-oauth20`.

Reply:
ok